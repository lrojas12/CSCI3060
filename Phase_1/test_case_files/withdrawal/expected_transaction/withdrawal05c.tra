10 Luisa                00020          S 
00 Luisa                00020          S 