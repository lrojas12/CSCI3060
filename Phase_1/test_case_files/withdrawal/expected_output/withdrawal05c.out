Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
Account holder’s name:
You are currently logged in as Luisa
Bank Account Number: 00020
Balance: $99999.99
Transaction Payment Plan: Non-student
Status: Active
Enter account number:
ERROR: Newly created accounts may not deposit funds. Please try again in 24 hours.
You have successfully logged out of your account.